src/lib/gerrit/gerritAPI/api.ts

```diff
@@ -608,6 +609,18 @@ export class GerritAPI {
 		return map;
 	}
 
+	public async getTopic(
```

Comment:
Don't think this is needed, the topic is returned on the `change` when we fetch the change from the Gerrit API. I forgot to add it to the types and to the `GerritChange` class though. So if you could do this:
* In `src/lib/gerrit/gerritAPI/types.ts` add `topic` as an optional string to `GerritChangeDetailResponse`
* In `src/lib/gerrit/gerritAPI/gerritChange.ts` add `topic` as a public property initialized from `reponse.topic` in the constructor.

Then instead of needing to use `getTopic` you can just use `change.topic` :)